src/test/groovy/com/gradle/BaseInitScriptTest.groovy

```diff
@@ -43,7 +43,7 @@ abstract class BaseInitScriptTest extends Specification {
     ]
 
     static final List<TestGradleVersion> CONFIGURATION_CACHE_GRADLE_VERSIONS =
-        [GRADLE_7_X, GRADLE_8_0, GRADLE_8_X].intersect(ALL_GRADLE_VERSIONS)
+        [GRADLE_8_0, GRADLE_8_X].intersect(ALL_GRADLE_VERSIONS)
```

Comment:
Why was `GRADLE_7_X` removed?

Reply:
The recent CCUD release broke configuration-cache support with Gradle 7.x.

The configuration-cache support in Gradle 7.x was experimental, and I think anyone using configuration-cache is very likely also using a more recent version of Gradle.

As such, I didn't think it was worthwhile spending time fixing config-cache compatibility just for Gradle 7.6.4.

Reply:
I've removed this change. The test will now fail with CCUDGP v2.2, and will require a v2.2.1 to fix.